testdata/jsight_0.3/ser-to-openapi/req.japi.schema_notation/req.japi.schema_notation.jsight_0.3/jsight_notation_in_directive_type.jst

```diff
@@ -0,0 +1,25 @@
+JSIGHT 0.3
+
+TYPE @typeWithJsightAnnotation_object jsight
+  {
+    "id": 1
+  }
+
+TYPE @typeWithJsightAnnotation_string jsight
+  "just a string"
+
+TYPE @typeWithJsightAnnotation_float jsight
+  1.2345
+
+TYPE @typeWithJsightAnnotation_array jsight
+  [ 1, 2, 3 ]
+
+TYPE @typeWithJsightAnnotation_null jsight
+  null
+
+GET /resource/with/types
+  200
+    @typeWithJsightAnnotation_object | @typeWithJsightAnnotation_string | @typeWithJsightAnnotation_float 
```

Comment:
It is desirable to use JSight with the capital "S" in camelCase format.

There are only two proper ways of writing `JSight`:
- `jsight`
- `JSight`

Reply:
Fixed in this PR.